.cascade-code/Chapman.edu/_cascade/templates/home_page_v3.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html class="no-js" lang="en">
+    <head>
+        <meta charset="utf-8"/>
+        <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible"/>
+        <title>
+            <system-page-title/>
+        </title>
+        <system-page-meta-keywords/>
+        <system-page-meta-description/>
+        <system-page-meta-author/>
+        <system-region name="OG_TAGS"/>
+        <system-region name="META VIEWPORT"/>
+        <!-- Jquery, Typkit, Common Assets -->
+        <script src="//use.typekit.net/eyn5jyy.js" type="text/javascript"></script>
+        <script type="text/javascript">try{Typekit.load();}catch(e){}</script>
+        <system-region name="JQUERY"/>
+        <system-region name="CASCADE ASSETS"/>
+        <system-region name="ADDITIONAL HEAD"/>
+        <system-region name="GOOGLE_ANALYTICS"/>
+    </head>
+    <body>
```

Comment:
Notice I removed all the alternative conditional body tags for old versions of IE we no longer support. Removed markup can be seen here:

- https://github.com/chapmanu/cascade-assets/pull/288/commits/c96a9d97f43d9b5b40bd242098a32b8ae38a0412